test/unit_tests/test_Sources_opt.jl

```diff
@@ -162,5 +162,12 @@ par = (; pdists = [ExponentialPrimitiveParticleDistribution(1.0, 1.0)], vel = [(
 
 ## Condensation.jl
 # Condensation moment tests
-par = (; pdists = [ExponentialPrimitiveParticleDistribution(1.0, 1.0)], ξ = 1e-6)
-@test_opt get_cond_evap(0.01, par)
+pdists = (ExponentialPrimitiveParticleDistribution(1.0, 1.0),)
+ξ = 1e-6
+s = 0.01
+@test_opt get_cond_evap(pdists, s, ξ)
+@test 144 >= @allocated get_cond_evap(pdists, s, ξ)
+# TODO allocation tests fail on the first run?
```

Comment:
Nat mentioned this on slack ([here](https://climate-machine.slack.com/archives/C0702CW3W3A/p1713392456011899?thread_ts=1713386414.278969&cid=C0702CW3W3A)), but `@allocated function()` includes allocations from compilation the first time `function()` is called. So I think it's expected that `alloc_limit >= @allocated function()` would fail if `function()` hasn't been called yet